ledger-service/http-json/BUILD.bazel

```diff
@@ -65,11 +65,22 @@ da_scala_binary(
     deps = [":http-json"] + http_json_deps,
 )
 
+genrule(
+    name = "integration-test-static-content",
+    srcs = ["src/test/integration-test-static-content/dummy.txt"],
+    outs = ["integration-test-static-content"],
+    cmd = """
+    mkdir -p $(location integration-test-static-content)
+    cp -L $(SRCS) $(location integration-test-static-content)
+    """,
+)
```

Comment:
Having `genrule`s creating directories is generally [not recommended](https://docs.bazel.build/versions/master/be/general.html#general-advice):
> Avoid creating symlinks and directories. Bazel doesn't copy over the directory/symlink structure created by genrules and its dependency checking of directories is unsound.

That's likely the cause for the CI failure on Windows.

Could `static-content` be extended to accept a zip archive in some form? Then you could zip those sources up using `@bazel_tools//tools/zip:zipper`. There are some examples in the repo, e.g. [here](https://github.com/digital-asset/daml/blob/99ea93168d84d456060bbdad4d7922e19eacf4f8/language-support/scala/codegen/codegen.bzl#L30).